### DISCUSSION TITLE
Demo of db2graphql + gqty

### DISCUSSION THREAD

**Discussion by taviroquai** [gqty-dev/gqty#1496] [Show and tell]
Hi guys,

First of all, congratulations for GQty!

I was playing around with **db2graphql** and with **GQty** and together they are amazing.
I almost can fetch nested data from a monster relational database in a snap!

**db2graphql** generates a Graphql schema and resolvers from an existing relational database.

Take a look a the repo demo:
https://github.com/taviroquai/demo_db2graphql_gqty

Cheers,

**Comment by taviroquai** [comment]
Let me send you screenshots of relevant parts.

![image](https://user-images.githubusercontent.com/498409/217697477-6ab3a9fc-4092-4a2d-a398-212bea397170.png)

![image](https://user-images.githubusercontent.com/498409/217697552-7fcb28b0-8d1c-4a05-97c0-acf8634502e0.png)

![image](https://user-images.githubusercontent.com/498409/217697630-983038a8-9cd3-4efc-82a1-fd3639323cce.png)

**Reply by vicary** [reply]
Great work there, love to see another DB solution!
